source/comp/markv.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2018 Google Inc.
```

Comment:
The company should be Google LLC.

Reply:
```
Development/spirv-tools/SPIRV-Tools % gg "Google LLC"|wc -l
138
Development/spirv-tools/SPIRV-Tools % gg "Google Inc"|wc -l
298
```

Reply:
Looks like a larger cleanup is needed here.